packages/scaffold/src/partials/w3m-account-auth-button/index.ts

```diff
@@ -0,0 +1,56 @@
+import { customElement } from '@web3modal/ui'
+import { LitElement, html } from 'lit'
+
+import {
+  ConnectorController,
+  RouterController,
+  StorageUtil,
+  type SocialProvider
+} from '@web3modal/core'
+
+@customElement('w3m-account-auth-button')
+export class W3mAccountAuthButton extends LitElement {
+  // -- Render -------------------------------------------- //
+  public override render() {
+    const type = StorageUtil.getConnectedConnector()
+    const authConnector = ConnectorController.getAuthConnector()
+
+    if (!authConnector || type !== 'AUTH') {
+      return null
```

Comment:
Do we need to set the `display: none`? When the `w3m-account-auth-button` rendered in a parent and if there is a `gap` between elements. Not setting `display: none` causing weird empty spaces between elements